app/containers/GuideProfilePage/index.js

```diff
@@ -36,7 +49,8 @@ export class GuideProfilePage extends React.Component { // eslint-disable-line r
 
               <div className="d-lg-none">
                 {/* Mobile/tablet view */}
-                mobile header
+                {/* Currently using dummy profile */}
+                <ProfileHeader guide {...sampleProfile} />
```

Comment:
Can you put it on desktop too?

Reply:
NP; no styling for desktop is set up, though.